src/NotepadNext/docks/SearchResultsDock.ui

```diff
@@ -30,6 +30,13 @@
     <property name="bottomMargin">
      <number>0</number>
     </property>
+    <item>
+     <widget class="QPushButton" name="pb_copyResults">
```

Comment:
I know I'm being overly nit-picky but something like `btnCopyResults` would be more consistent with the rest of the UI elements in the other parts of the application.

Reply:
You are quite right again, I should have looked ahead and followed the design of the existing code, not just throwing my own style in there. Attention to detail!!